src/utils/handlers/general/searchTrack.ts

```diff
@@ -115,6 +118,11 @@ export async function searchTrack(
                                 )
                             );
 
+                            if (songIndex) {
+                                const temp = tracks.splice(parseInt(songIndex) - 1, 1)[0];
+                                tracks.unshift(temp);
+                            }
+
```

Comment:
Handle when index is invalid
```suggestion
                            if (songIndex) {
                                const temp = tracks.splice(parseInt(songIndex) - 1, 1)[0];
                                if (temp) tracks.unshift(temp);
                            }

```

Reply:
added verification to check if songIndex < 101 (queue seems to have a maximum of 100)
- if index is between 1 and 100 it will do the splice
- if not it will assign null so it fails the next condition, and plays from track 1
- was getting an error on build if I just left `const temp = tracks.splice(parseInt(songIndex) - 1, 1)[0];` since `temp` would always be True `error  Unnecessary conditional, value is always truthy`